src/firefly/js/charts/ChartUtil.js

```diff
@@ -519,6 +530,46 @@ export function setupTableWatcher(chartId, ts, idx) {
         uniqueId(`ucd-${ts.tbl_id}-trace`)); // watcher id for debugging
 }
 
+/**
+ * Get feedback as boolean on whether chart is fully loaded or not
+ * @param chartId - chartId of the current chart
+ */
+export function isChartLoaded(chartId) {
+    const {fireflyData=[]} = getChartData(chartId);
+    const isLoading = fireflyData.every((e)=>get(e, 'isLoading'));
+    return !isLoading; //true when chart is fully loaded (isLoading: false)
+}
+
+const onChartLoad = (action, accept, cancelSelf) => {
```

Comment:
It turns out this is not needed.  This logic is not generic to be a util function.